CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## 2.27.1 (Unreleased)
+FEATURES:
+* **New Resource:** sumologic_azure_event_hub_log_source (GH-590)
```

Comment:
Is there a way to indicate that this feature is in Beta?

Reply:
I think we usually added description of `XXX is in beta` in markdown file: [website/docs/r/azure_event_hub_log_source.html.markdown](https://github.com/SumoLogic/terraform-provider-sumologic/pull/590/files#diff-0cd1b52b8e26bf7e5dc5bb98f301bba1e784b3698bbc2df8da64f8ae15d95ed1)